README.md

```diff
@@ -142,24 +150,68 @@ int/float getStat(string stat);
 * **Action:** None
 * **Response:** `true` if there is a wall to the left of the robot, else `false`
 
-#### `moveForward [N]`
+
+#### `moveFullForward [N]`
+* **Args:**
+  * `N` - (optional) The number of cells to move forward, default `1`
+* **Action:** Move the robot full-size forward the specified number of cells
+* **Response:**
+  * `crash` if `N < 1` or the mouse cannot complete the movement
+  * else `ack` once the movement completes
+
+#### `moveHalfForward [N]`
 * **Args:**
   * `N` - (optional) The number of cells to move forward, default `1`
-* **Action:** Move the robot forward the specified number of cells
+* **Action:** Move the robot half-size forward the specified number of cells
+* **Response:**
+  * `crash` if `N < 1` or the mouse cannot complete the movement
+  * else `ack` once the movement completes
+
+#### `moveEdgeForward [N]`
+* **Args:**
+  * `N` - (optional) The number of cells to move diagonal forward, default `1`
+* **Action:** Move the robot edge forward the specified number of cells from a cell's edge
 * **Response:**
   * `crash` if `N < 1` or the mouse cannot complete the movement
   * else `ack` once the movement completes
 
 #### `turnRight`
 * **Args:** None
-* **Action:** Turn the robot ninty degrees to the right
+* **Action:** Turn the robot ninty degrees to the right from a cell's center
 * **Response:** `ack` once the movement completes
 
 #### `turnLeft`
 * **Args:** None
-* **Action:** Turn the robot ninty degrees to the left
+* **Action:** Turn the robot ninty degrees to the left from a cell's center
 * **Response:** `ack` once the movement completes
 
+#### `turnRight45`
+* **Args:** None
+* **Action:** Turn the robot 45 degrees to the right from a cell's edge
+* **Response:** `ack` once the movement completes
+
+#### `turnLeft45`
+* **Args:** None
+* **Action:** Turn the robot 45 degrees to the left from a cell's edge
+* **Response:** `ack` once the movement completes
+
+#### `turnBack`
+* **Args:** None
+* **Action:** Turn the robot 180 degrees to the left from a cell's center
+* **Response:** `ack` once the movement completes
```

Comment:
This feels unnecessary, because the user could achieve this behavior on their own by just calling `turnLeft` or `turnRight` twice in a row. Is it absolutely essential to include it here? If not, just remove it.